gomint-api/src/main/java/io/gomint/scheduler/AsyncScheduler.java

```diff
@@ -0,0 +1,38 @@
+package io.gomint.scheduler;
+
+import java.util.concurrent.TimeUnit;
+
+public interface AsyncScheduler {
+
+    /**
+     * Run the runnable in another Thread.
+     *
+     * @param runnable which should be executed
+     * @return the created and scheduled Task
+     */
+    Task executeAsync(Runnable runnable);
```

Comment:
Why are methods suffixed with Async? This alreay is AsyncScheduler, can we remove that suffix?

Reply:
The SyncScheduler extends this one.